stories/Card.stories.mdx

```diff
@@ -0,0 +1,104 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
+import { linkTo } from '@storybook/addon-links';
+import { Card, Header, Footer, Text } from '../src'; // components you used in the story
+
+
+<Meta title="Layout/Card" component={Card} />
+
+# Card
+
+The card will be used as a general interface to provide general structure to a UI component.
+It comes with very basic styling, but has a ton of customization. It includes options for headers and footers.
+The card component is made in such a way that it can be used as a container for other components.
+
+## Functionality
+
+Functionally these cards are very similar to the HTML `<div>` element. 
+The main purpose of a card is to provide information about a single piece of information.
+
+<Canvas>
+	<Story name="Standard Card" args={{ children: "This is card component" }}>
+		{(args) => <Card {...args} />}
+	</Story>
+</Canvas>
+
+## Setup
+
+Cards are setup in a way that they can be used as a container for other components. Think of it as a
+blank canvas with posibilities to add almost anything you like. So the way to use a card is to wrap
+it around any component that you want to display inside the card. Generally cards comes with a `header`
+then a body and then a `footer`.
+
+Here is a simple example of how to use a card.
+
+<Canvas>
+	<Story name="Complete-Card">
+        <Card>
+            <Header>This is the header</Header>
+            This card wraps a header and a footer.
+            <Footer>This is the footer</Footer>
+        </Card>
+	</Story>
+</Canvas>
+
+That's it we have a card with a `header` and a `footer` and it look great :)
+But what if you want to customize the card?
+No issues at all, you can customize the card and the components inside it as you wish.
+
+Let's customize this card to make  it look more elegent.
+
+<Canvas>
+	<Story name="Custom-Card">
+		<Card style={{ height: 200, width: 300 }}>
+        <Header>
+            <Text header={2} bold>
+                Custom Card
+            </Text>
+        </Header>
+        <Text bold color="gray">
+            What is this?
+        </Text>
+        <Text margins> This is a glorified div that supports headers and footers</Text>
+        <Footer style={{ borderTop: '1px solid lightgray' }}>
+            <Text italic style={{ marginTop: 10 }}>
+                per request :)
+            </Text>
+        </Footer>
+        </Card>
+	</Story>
+</Canvas>
+
+### `Card With Header`
+
+Since the card component works as a container for other components, 
+you add a `header` component to it which will work as a title for your content in the body.
```

Comment:
Make sure you capitalize ***H***eader and ***F***ooter. Also you should say that the `Header` component will always be on top of the rest of the children and the `Footer` component will always be on bottom. This is to facilitate layout.